src/side-navigation/styles.scss

```diff
@@ -94,6 +90,20 @@
   padding-block: 0;
   padding-inline: 0;
   list-style: none;
+
+  @supports selector(:has(*)) {
```

Comment:
Do the `@supports` and `@supports not` selectors exclude each other? If so, could we simplify this by keeping only the `@supports not` and removing the additional rules and stylelint suppressions?

Reply:
It would be preferred to use the `:has` selector here and keep the margins all on the outer `.list-container`, so we don't end up with double margins around sections like we do sometimes today. The `@supports not` is only kept for backwards compatibility, ideally once Safari 15.1, 15.2, and 15.3 supports the `:has` selector, the `@supports not` lines can be removed.